### ISSUE TITLE
Demo seem to not work

### ISSUE THREAD

**Issue by JFDionne** [Merve40/ptt#2]
Hi, I'm trying your demo, I'm using Chrome Windows and Android Chrome for testing it, but when I do press push to talk I don't hear anything.

**Comment by Merve40** [comment]
Hello JDionne. In order for me to help you, I need to know the following answers:
* Are you deploying your own server or using the demo website?
* Which Android Version are you using?
* Are you connecting to `https`?
* Are you granting the permissions for microphone on both Desktop and Android?

**Comment by JFDionne** [comment]
Hi. 

Using your demo webpage. 
Yes I grant microphone permission. 
I use https. 
For android I use chrome with android 7. But i tried to open chrome on windows on 2 computer and I can’t hear anything.

**Comment by Merve40** [comment]
Ok. When testing with Android, I recommend using the app instead of the browser, since the overall performance is better. Which Windows Version(s) and Chrome Desktop Version(s) are you using?

**Comment by JFDionne** [comment]
Windows 10
Chrome: 78.0.3904.108

When I press button on windows 1 I don't have any error, but if I press button on windows 2, I got those error on windows 1:
![image](https://user-images.githubusercontent.com/1706002/69470836-3e864180-0d67-11ea-8dd8-207d5140c523.png)

**Comment by Merve40** [comment]
I did fix some minor bugs, however I am not sure if they will fix your problem.     

This error: `Uncaught Error: Argument should be an AudioBuffer instance.` can be ignored, as I think it always mistakenly shows whenever an audio is being written to buffer. It must be a bug in the API, that I am using.

Unfortunately, I tried to recreate the issue with **Windows 10 & Chrome**, but everything works fine. 
I tested on **Windows & Linux**, with both **Firefox & Chrome**, there were no problems.
I suggest that you try out Firefox too.
